dev-support/hbase_nightly_yetus.sh

```diff
@@ -67,6 +64,7 @@ YETUS_ARGS=("--whitespace-tabs-ignore-list=${WHITESPACE_IGNORE_LIST}" "${YETUS_A
 YETUS_ARGS=("--sentinel" "${YETUS_ARGS[@]}")
 YETUS_ARGS=("--branch=${BRANCH_NAME}" "${YETUS_ARGS[@]}")
 YETUS_ARGS=("--tests-filter=${TESTS_FILTER}" "${YETUS_ARGS[@]}")
+YETUS_ARGS=("--ignore-unknown-options=true" "${YETUS_ARGS[@]}")
```

Comment:
What's still unknown?

Reply:
JDK/Hadoop specific stages do not run all plugins so whitespace, author and shelldocs are not recognized options there.

Reply:
Okay that makes sense.